src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-651.mdx

```diff
@@ -0,0 +1,18 @@
+---
+subject: Android agent
+releaseDate: '2022-04-13'
+version: 6.5.1
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_6.5.1.zip'
+---
+
+## Fixed in this release
+
+* Resolved a memory leak that occurred in long-lived apps.
```

Comment:
Please add a reference to where that leak occurred, and what was leaked

Reply:
Provided more details